### ISSUE TITLE
BigQuery `ST_GEOGFROMGEOJSON` breaks parser when using `make_valid` parameter

### ISSUE THREAD

**Issue by daaain** [sqlfluff/sqlfluff#1083]
https://cloud.google.com/bigquery/docs/reference/standard-sql/geography_functions#st_geogfromgeojson

## Expected Behaviour

Pass

## Observed Behaviour

```sh
L:  25 | P:  89 | L027 | Unqualified reference 'make_valid' found in select with
                       | more than one referenced table/view.
L:  25 | P:  99 |  PRS | Found unparsable section: ' => true'
```

## Steps to Reproduce

Using the optional `make_valid` parameter in `ST_GEOGFROMGEOJSON` breaks the parser.

For example:

```sql
st_geogfromgeojson(lsoa.geography, make_valid => true)
```

## Version

```sh
app@27fc52961e2b:/app$ sqlfluff --version
sqlfluff, version 0.5.6
app@27fc52961e2b:/app$ python --version
Python 3.7.10
```

## Configuration

```
[sqlfluff]
templater = dbt
dialect = bigquery
exclude_rules = L031

[sqlfluff:rules]
tab_space_size = 2
max_line_length = 100

[sqlfluff:rules:L030]
capitalisation_policy = lower

```

**Comment by barrywhart** [comment]
Here's a full `SELECT` query that produces the issue:
```
SELECT ST_GEOGFROMGEOJSON('{"type":"LineString","coordinates":[[1,2],[4,5]]}', make_valid => true)```
